### DISCUSSION TITLE
[Support]: Got 404 when deleting exported files

### DISCUSSION THREAD

**Discussion by lorenzomartini1** [blakeblackshear/frigate#11493] [General Support]
### Describe the problem you are having

I've exported some clips with "Exports" feature.
When I try to delete clips, sometimes it gives me `404 Not Found` with this response:

```
{
    "message": "porta_vista_nord_2024_05_23_12_34__2024_05_23_12_36.mp4 not found.",
    "success": false
}
```
The strange thing is that I've exported that clip less then 5 minutes ago, so it's not an old one.

### Version

0.13.2-6476F8A

### Frigate config file

```yaml
na
```

### Relevant log output

```shell
na
```

### FFprobe output from your camera

```shell
na
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

na

**Comment by NickM-27** [comment]
This is a known issue, fixed in 0.14
